src/style.css

```diff
@@ -1 +1,50 @@
 /* add styles here */
+body {
+  display: flex;
+  margin: 0;
+  background-color: gray;
+}
+
+.display {
+  display: flex;
+  margin: 0 auto;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Here you have a margin set to '0 auto', which is fine for horizontally centering, but make sure you're not adding unnecessary vertical margins elsewhere that could lead to margin collapse.